src/reactviews/pages/ExecutionPlan/properties.tsx

```diff
@@ -0,0 +1,631 @@
+                        {`\u200b\t`.repeat(item.level * 6)}
+                        {item.children.length > 0 && (
+                            <Button
+                                size="small"
+                                className={classes.chevronButton}
+                                icon={
+                                    openedButtons.includes(item.name) ? (
+                                        <ChevronDown20Regular />
+                                    ) : (
+                                        <ChevronRight20Regular />
+                                    )
+                                }
+                                onClick={() =>
+                                    handleShowChildrenClick(
+                                        item.name,
+                                        item.children,
+                                    )
+                                }
+                            />
+                        )}
+                        {item.name}
+                    </TableCellLayout>
+                ),
+            }),
+            createTableColumn<ep.ExecutionPlanPropertyTableItem>({
+                columnId: "value",
+                renderHeaderCell: () => VALUE,
+                renderCell: (item) => (
+                    <TableCellLayout truncate className={classes.textContainer}>
+                        {item.value}
+                    </TableCellLayout>
+                ),
+            }),
+        ];
+
+    return (
+        <div
+            id="propertiesPanelContainer"
+            className={classes.paneContainer}
+            style={{
+                background: utils.background(executionPlanState!.theme!),
+                borderLeft: `2px solid ${utils.tableBackground(executionPlanState!.theme!)}`,
+            }}
+        >
+            <div
+                className={classes.propertiesHeader}
+                style={{
+                    background: utils.background(executionPlanState!.theme!),
+                }}
+            >
+                <div>{PROPERTIES}</div>
+                <div>
+                    <Button
+                        className={classes.dismissButton}
+                        style={{
+                            background: utils.background(
+                                executionPlanState!.theme!,
+                            ),
+                        }}
+                        onClick={() => setPropertiesClicked(false)}
+                        icon={<Dismiss12Regular />}
+                    />
+                </div>
+            </div>
+            <div
+                className={classes.nameContainer}
+                style={{
+                    background: utils.tableBackground(
+                        executionPlanState!.theme!,
+                    ),
+                }}
+            >
+                {name}
+            </div>
+            <Toolbar className={classes.toolbar}>

```

Comment:
What if set the size of toolbar to `small`.  That will reduce all the extra spacing around the toolbar icons. 
Same for the input box.